app/controllers/concerns/idv_session.rb

```diff
@@ -1,6 +1,17 @@
 module IdvSession
   extend ActiveSupport::Concern
 
+  included do
+    before_action :sp_context_needed?
```

Comment:
Because this is pretty broad, I just want to double-check we want to verify in all cases.

One example: When we send user an email including a link to reverify, will there be an SP associated?

https://github.com/18F/identity-idp/blob/b7298bc6c14a528b0d5aee5b13fb9850957d67b4/app/views/user_mailer/confirm_email_and_reverify.html.erb#L31-L34